OSDOCS CQA AUTH-4 Service Accounts and Cloud Credential Operator I (#115126)
openshift/openshift-docs: authentication/managing_cloud_provider_credentials/cco-mode-manual.adoc

:_mod-docs-content-type: ASSEMBLY
[id="cco-mode-manual"]
= About manual mode with long-term credentials for components
include::_attributes/common-attributes.adoc[]
:context: cco-mode-manual

toc::[]

[role="_abstract"]

> A user can manage your cloud credentials instead of the Cloud Credential Operator (CCO) by setting the Operator to manual mode.

Reviewer: ```suggestion
You can manage your cloud credentials instead of the Cloud Credential Operator (CCO) by setting the Operator to manual mode.
```
Makes the ref match, but also the user is the cluster admin so we can probably just say "You" (unless I am missing some rule - very possible)
